mk/spksrc.service.installer

```diff
@@ -145,6 +154,41 @@ set_syno_permissions ()
     fi
 }
 
+# Sets recurivse permissions using chown
+set_unix_permissions ()
+{
+    DIRNAME=$1
+    if [ -n "${EFF_USER}" ]; then
+        echo "Granting '${EFF_USER}' unix permissions on ${DIRNAME}" >> ${INST_LOG}
+        if [ $SYNOPKG_DSM_VERSION_MAJOR -lt 6 ]; then
+            chown -R ${EFF_USER}:root "${DIRNAME}" >> $INST_LOG 2>&1
```

Comment:
@Safihre @ymartin59 I have continued to debug the upgrade problems on DSM5.2 systems that I ran into yesterday (packages did not start anymore after upgrade due to permission issues). I suspect the culprit is that the group is set to `root` here rather than `users` for DSM lt 6. Do you agree? If so, we probaby have to re-build all packages which were built since.

Reply:
Most test packages have only built for DSM 6... Group "users" is used for share folders accessible from other processes. For a package specific folder, group "root" should be far enough as owner matches service user.

Reply:
Agreed with @ymartin59, but unfortunately in the case of Tvheaded everything is mixed. Package and "user" folders blend together.

But @m4tt075, do you need one use `set_syno_permmissions` at all in your case? Since everything stays in the Package directory, do you need to call any permissions command at all?

Reply:
Isn't really impossible to configure TVH download location to a share folder as other applications ? I think that may be the key. If download locations are "hardcoded", it is still possible to create symbolic link from expected/original locations to DSM share folder(s)

Reply:
OK, thanks for clarifying. @Safihre I agree with you. Unix permissions need to be sufficient for all folders within `@appstore`. I have been able to implement that approach successfully for DSM6.1 now. I have tested clean installs, upgrading by itself and upgrading from the currently published package. But it does not work on DSM 5.2, when upgrading from the currently published package. In that version, ACL permissions were still set on the `target` and traversal folders. Is there a way to "remove" those ACL permissions again?

Reply:
@m4tt075 Why are ACL permissions applied? Only if `set_syno_permissions` is called this happens, but that shouldn't be needed at all, right?